### DISCUSSION TITLE
missing Input floating labels customization in PrimeOne-3.0.1

### DISCUSSION THREAD

**Discussion by dexterial** [primefaces/community#3122] [Figma UI Kit]
[https://primevue.org/floatlabel/](https://primevue.org/floatlabel/) is there a roadmap for this implementation in the figma kit ?

**Comment by onursenture** [comment]
Hello, the Float Label is not a separate component but is available as a property under the InputText component. 

<img width="1424" alt="image" src="https://github.com/user-attachments/assets/65eb5b4a-1ba9-46ef-ab5e-ed6ac3e06bff">

**Reply by dexterial** [reply]
While that is true, the variants are missing. 
![Screenshot_20241130-220532_Chrome](https://github.com/user-attachments/assets/209add08-9844-4393-a1f1-59def471f6a4)

**Reply by onursenture** [reply]
Hello, Yes, thank you for your feedback. We've noted your suggestions to add new float label variants and iftalabel in the next version.
